parallel-install/example/example.go

```diff
@@ -30,8 +32,9 @@ func main() {
 
 	log := logger.NewLogger(*verbose)
 
-	if kubeconfigPath == nil || *kubeconfigPath == "" {
-		log.Fatal("kubeconfig is required")
+	kubeConfigManager, err := config.NewKubeConfigManager(kubeconfigPath, kubeconfigContent)
```

Comment:
is it necessary to expose this manager to our clients? in my opinion it should be hidden and resolved under the hood despite what clients set: kubeconfigPath / kubeconfigContent

Reply:
It is not. Now `KubeconfigManager` works on both parameters under the hood in every component that needs kubeClient or dynamicClient.